<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_course</class>
 <widget class="QDialog" name="Add_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1" colspan="4">
        <widget class="QLineEdit" name="Distance"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Расстояние, в милях</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QLineEdit" name="SpeedShip"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Средняя скорость судна, в узлах</string>
         </property>
        </widget>
       </item>
       <item row="5" column="4">
        <widget class="QPushButton" name="Update">
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="5">
        <widget class="QTableView" name="TableViewSites">
         <property name="editTriggers">
          <set>QAbstractItemView::AnyKeyPressed</set>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="DeleteSite">
         <property name="text">
          <string>Удалить участок</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="AddSite">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Добавить участок</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Идентификатор маршрута(Название)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="4">
        <widget class="QLineEdit" name="NameCourse"/>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ApplyButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelButton">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" rowspan="2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Количество топлива в литрах</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="CountFuel">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="Oil_repair">
       <property name="text">
        <string>Была ли произведена замена масла</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="title">
        <string>Тип масла</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="Oil_1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Малой продолжительности</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Oil_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Средней продолжитлельности</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="Oil_3">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Большой продолжительности</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>2</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
       <zorder>Oil_1</zorder>
       <zorder>verticalSpacer</zorder>
       <zorder>Oil_2</zorder>
       <zorder>Oil_3</zorder>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="FuelRepair">
       <property name="text">
        <string>Была ли произведена заправка топлива</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
